catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -545,4 +552,11 @@ private QueryRequestImpl getQuery(Filter filter, int index, int pageSize) {
                 TimeUnit.MINUTES.toMillis(1)), new HashMap<>());
     }
 
+    public void setStorageProvider(StorageProvider storageProvider) {
```

Comment:
❓ Do we need `setStorageProvider` and `setJarSigner`? Doesn't look like they are used anywhere.